docs/DeveloperGuide.md

```diff
@@ -333,9 +422,139 @@ should be able to accomplish most of the tasks faster using commands than using
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Insurance4Insurance**: Name of the product.
 * **I4I**: Short for Insurance4Insurance, the name of the product.
+* **Active Client**: A client not in the archive.
+* **Active List** / **Active Client List**: List of clients not in the archive.
+* **Active Mode**: The mode which allows users to view the active list.
+* **Archive Mode**: The mode which allows users to view the archive.
+
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Appendix: Instructions for manual testing**
 
-Coming soon.
+Given below are instructions to test the app manually.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
+testers are expected to do more *exploratory* testing.
+
+</div>
+
+
+### Launch and shutdown
+
+1. Initial launch
+
+   1. Download the jar file and copy into an empty folder.
+
+   2. Launch the app by using the `java -jar` command (do not use double-clicking). Expected: Shows the GUI with a set of sample clients. The window size may not be optimum.
```

Comment:
Not sure is this ... what is written on the AB3? If so then we will leave it >< Otherwise we should mention that we can double click it.

Reply:
I followed the PE instructions for this one; they specifically told us not to double click